cinema/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from cinema.models import Movie
```

Comment:
Similar to the previous comment, ensure there is a newline after the import statement for better readability.